test/integration/MongooseAdapterSpec.js

```diff
@@ -72,4 +86,18 @@ describe('MongooseAdapterIntegration', function () {
       .catch(err => done(err))
     ;
   });
+
+  /* eslint-disable no-underscore-dangle */
+  it('allows to pass mongo ObjectId as defalt attibute', function (done) {
+    mongoUnavailable ? this.skip() : null;
```

Comment:
It could be cleaner as the `null` is pointless, either of these seems better
```js
if(mongoUnavailable) this.skip();
```
```js
mongoUnavailable && this.skip();
```